src/slack_api_util.ts

```diff
@@ -239,7 +240,9 @@ export async function sendMessage(
 
       try {
         await DbApiUtil.logMessageToDb(databaseMessageEntry);
-        await DbApiUtil.updateThreadStatusFromMessage(databaseMessageEntry);
```

Comment:
Question about this earlier implementation: SlackApiUtil.sendMessage is used for sending non-voter messages to Slack as well. If a volunteer dropdown is changed, for example, an update is sent to the thread. In those cases, is the thread status updated?

Reply:
Yeah, good point.  I think this needs a bigger clean-up.